src/agent/io/sourcemapper.ts

```diff
@@ -34,6 +35,10 @@ export interface MapInfoInput {
   // process's working directory).
   outputFile: string;
 
+  // The path of the original source file. In the example above, this will be
```

Comment:
Please specify what this path is relative to.

Reply:
Done.